gcc/rust/lang.opt

```diff
@@ -83,6 +83,10 @@ frust-cfg=
 Rust Joined RejectNegative
 -frust-cfg=<name>             Set a config expansion option
 
+frust-edition=
+Rust Joined RejectNegative
+-frust-edition=<year>             Choose which edition to use when compiling rust code
+
```

Comment:
You're right! Thank you. This is much cleaner :)

Reply:
Yeah that looks like a good solution

Reply:
Done in [46e0068](https://github.com/Rust-GCC/gccrs/pull/1091/commits/46e0068fc0128207ac69bde5cb53d7eeb6943cbb) and I co-authored you @liushuyu :) Feel free to re-review!

Reply:
> Done in [46e0068](https://github.com/Rust-GCC/gccrs/pull/1091/commits/46e0068fc0128207ac69bde5cb53d7eeb6943cbb) and I co-authored you @liushuyu :) Feel free to re-review!

Wow. Thank you! Since I provided that suggestion, I probably can't comment too much on this :smile:.

If you are happy with the current implementation, I'd say ping someone else to review this.